lib/service/Service.ts

```diff
@@ -187,6 +187,14 @@ class Service extends EventEmitter {
     await this.pool.unban(args);
   }
 
+  /**
+   * Get node info.
+   */
+  public getNodeInfo = async (args: {nodePubKey: string}) => {
```

Comment:
I'm actually surprised this didn't throw a lint error (I'll look into that), but every where else we have a space after the open bracket `{` and before the close bracket `}` if you don't mind changing that.

Reply:
Yeah, I don't know why that happend normally the `npm run lint` saves me from such mistakes.